test/compilable/b17111.d

```diff
@@ -0,0 +1,15 @@
+import std.stdio : readf, writeln;
+alias TestType = ubyte;
+
+void main()
+{
+    TestType a,b,c;
+    readf("%s %s %s ", &a, &b, &c);
```

Comment:
This test case can be rewritten to not use phobos. This is desirable to speed the compilation of the test suite, and to focus the test case onto just what the problem is, rather than bringing in heaps of irrelevant complication from phobos.